cmd/kubeseal/main.go

```diff
@@ -285,6 +291,34 @@ func main() {
 		panic(err.Error())
 	}
 
+	if *raw {
+		data, err := ioutil.ReadAll(os.Stdin)
+		if err != nil {
+			panic(err.Error())
+		}
+		ns, _, err := clientConfig.Namespace()
+		if err != nil {
+			panic(err.Error())
+		}
+		if ns == "" {
+			panic("Must provide the --namespace flag with --raw")
+		}
+		if *secretName == "" {
+			panic("Must provide the --name flag with --raw")
+		}
+		toEncrypt := make([]byte, base64.StdEncoding.DecodedLen(len(data)))
+		_, err = base64.StdEncoding.Decode(toEncrypt, data)
+		if err != nil {
+			panic(err.Error())
+		}
+		out, err := crypto.HybridEncrypt(rand.Reader, pubKey, toEncrypt, []byte(fmt.Sprintf("%s/%s", ns, *secretName)))
```

Comment:
we need to make this work in cluster-wide and namespace-wide modes. I'm in the process of refactoring that part anyway.